src/Workload/Microsoft.NET.Sdk.Maui/WorkloadManifest.in.json

```diff
@@ -282,9 +284,19 @@
       "kind": "library",
       "version": "@MAUI_GRAPHICS_VERSION@"
     },
-    "Microsoft.Maui.Sdk": {
+    "Microsoft.Maui.Sdk-@MAUI_DOTNET_VERSION@": {
       "kind": "sdk",
-      "version": "@VERSION@"
+      "version": "@VERSION@",
+      "alias-to": {
+        "any": "Microsoft.Maui.Sdk"
+      }
+    },
+    "Microsoft.Maui.Sdk-@MAUI_PREVIOUS_DOTNET_VERSION@": {
+      "kind": "sdk",
+      "version": "@MAUI_PREVIOUS_DOTNET_RELEASED_NUGET_VERSION@",
+      "alias-to": {
+        "any": "Microsoft.Maui.Sdk"
+      }
     },
```

Comment:
The aliased packs will pull the nugets for the correct versions. As a result, 2 versions of the Microsoft.Maui.Sdk package is installed: net7 and net6. This allows us in the targets to basically pick which one we want at build time.